src/main/java/uk/gov/companieshouse/model/Constants.java

```diff
@@ -14,7 +14,8 @@ private Constants() {
    public static final String PAYMENT_RESOURCE_KIND = "dissolution-request#dissolution-request";
    public static final String PAYMENT_DESCRIPTION = "Dissolution application";
    public static final String PAYMENT_DESCRIPTION_IDENTIFIER = "Dissolution application";
-   public static final String PAYMENT_PRODUCT_TYPE = "Dissolution application";
+   public static final String PAYMENT_PRODUCT_TYPE_DS01 = "16032";
```

Comment:
i think the dissolution api should return `ds01` or `llds01` - it is up to the payment reconciliation consumer to map the string to a code.


I think the Payment Response object should update the type of the Product Type field to be `ApplicationType`